env.development.local

```diff
@@ -0,0 +1,2 @@
+REACT_APP_SERVER=http://localhost:5000
+REACT_APP_FE=http://localhost:3000
```

Comment:
this file should not be visible here. 
You need to add it to the gitignore:

# misc
.DS_Store
.env.local
.env.development.local
.env.test.local
.env.production.local

Reply:
rather than doing this, I recommend renaming the file instead of .env to be be .env.development.local that file is set up so that the variables with be exported locally during development